packages/components/src/List/Toolbar/ColumnChooser/Content/ColumnChooserContent.component.js

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect } from 'react';
+
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import RichLayout from '../../../../RichTooltip/RichLayout';
+import theme from './ColumnChooser.scss';
+import { useColumnChooserManager } from '../hooks';
+import { DefaultHeader, DefaultBody, DefaultFooter } from './DefaultColumnChooser.components';
+
+export default function ColumnChooserContent({
+	body,
+	columns,
+	footer,
+	header,
+	id,
+	lockedLeftItems,
+	onClose,
+	submitColumnChooser,
+	t,
+}) {
+	const {
+		onChangeVisibility,
```

Comment:
Sorry for this one, but the naming kind of bother me :D
`onChangeVisibility` is not correct english readable vs `onVisibilityChange`

It's ok to keep it that way, but if you want to change :D
(this is valid for all the actions name except onSelectAll I think)

Reply:
The idea was to have the concatenation = event name + attribute.
Don't mind to change but having some rules will be nice and will make code readibility simplier.